contrib/babelfishpg_common/src/babelfishpg_common.h

```diff
@@ -77,6 +77,7 @@ typedef struct common_utility_plugin
 										  bool *isBaseDec, bool *isBaseBin,
 										  bool *isBaseDate, int *variantHeaderLen);
 	Oid			(*lookup_tsql_datatype_oid) (const char *typestr);
+	void	   	*(*resolve_pg_type_to_tsql) (Oid oid);
```

Comment:
If we are unable to return char* then instead of returning char* to void *, lets use argument to retrieve the name. Something like get_tsql_datatype_name(Oid oid, char **typname).

Reply:
Done